pkg/components/exporter/exporter.go

```diff
@@ -106,6 +108,74 @@ func NewDaemonSet(detail components.Detail, k *v1alpha1.Kepler) *appsv1.DaemonSe
 
 	bindAddress := "0.0.0.0:" + strconv.Itoa(int(deployment.Port))
 
+	exporterContainer := &corev1.Container{
```

Comment:
```suggestion
	exporterContainer := newExporterContainer(k)
```
Lets move all the creation to a `newExporterContainer()` which can be unit tested as well.